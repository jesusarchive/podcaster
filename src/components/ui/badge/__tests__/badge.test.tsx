import { render } from '@testing-library/react';
import React from 'react';

import Badge from '..';

describe('Badge', () => {
  it('should render the badge component', () => {
    const { container } = render(<Badge>Badge</Badge>);

    expect(container).toMatchSnapshot();
  });

  it('should render the badge component with a custom class name', () => {
    const { container } = render(<Badge className="custom-class">Badge</Badge>);

    expect(container.firstChild).toHaveClass('custom-class');
    expect(container).toMatchSnapshot();
  });
});
