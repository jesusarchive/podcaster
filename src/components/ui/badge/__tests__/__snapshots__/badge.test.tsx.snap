// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge should render the badge component 1`] = `
<div>
  <span
    class="badge"
  >
    Badge
  </span>
</div>
`;

exports[`Badge should render the badge component with a custom class name 1`] = `
<div>
  <span
    class="badge custom-class"
  >
    Badge
  </span>
</div>
`;
